client/src/app/app.ts

```diff
@@ -17,7 +17,7 @@ import { ManagingBodyService } from './managing-body/managing-body.service';
 @RouteConfig([
   { path: '/home',               component: HomeComponent,                name: 'Home', useAsDefault: true },
```

Comment:
Just look at your comments now, it make me a better programmer. thanks about that
start to work on it now